spi/identity-hub-spi/src/main/java/org/eclipse/dataspaceconnector/identityhub/models/MessageStatus.java

```diff
@@ -26,4 +32,33 @@ public class MessageStatus extends Status {
     private MessageStatus(int code, String detail) {
         super(code, detail);
     }
+
+    @JsonPOJOBuilder(withPrefix = "")
```

Comment:
Is a builder needed for this class? We're using builders when a data type has many attributes, this doesn't seem to be the case here.